packages/components/space/src/use-space.ts

```diff
@@ -57,7 +57,7 @@ export function useSpace(props: SpaceProps) {
       if (isNumber(size)) {
         val = size
       } else {
-        val = SIZE_MAP[size] || SIZE_MAP.small
+        val = SIZE_MAP[size || 'small'] || SIZE_MAP.small
```

Comment:
It seems that both are the same

```diff
- val = SIZE_MAP[size || 'small'] || SIZE_MAP.small
+ val = SIZE_MAP[size || 'small']
```

Reply:
if `size` is an invalid value, it will fallback to `small`